Features/Context/SubContext/Authentication.php

```diff
@@ -51,6 +51,7 @@ public function goToPlatformUiAndLogIn($username, $password)
 
     /**
      * @Given I am logged in as admin on PlatformUI
+     * @Given I am logged in as an Editor in PlatformUI
      */
     public function loggedAsAdminPlatformUi()
```

Comment:
Nitpick / OT: 
Not necessarily admin user, maybe `loggedInAsUserInPlatformUi()` ?
Also IMO since the trait is using vars from the context instance, at least document it?